src/main/java/net/masterthought/cucumber/json/Match.java

```diff
@@ -4,10 +4,14 @@
 
     // Start: attributes from JSON file report
     private final String location = null;
+    private final MatchArgument[] arguments = new MatchArgument[0];
```

Comment:
There's already an Argument class. It's in the support package, so it would possible to rename MatchArgument to Argument, but it might be confusing to have two classes with the same name. Just let me know what you prefer.